packages/docs-ui/src/components/SideNavigation/SideNavigationItem.tsx

```diff
@@ -0,0 +1,49 @@
+import { Badge, Bleed, ButtonLink, Inline } from 'braid-design-system';
+import React from 'react';
+
+type BadgeLabel = 'New' | 'Deprecated';
+
+const toneForBadge = (badgeLabel: BadgeLabel) => {
+  const toneMap = {
+    Deprecated: 'caution',
+    New: 'positive',
+  } as const;
+
+  return toneMap[badgeLabel];
+};
```

Comment:
This doesnt need to be a function call, would be simpler just as an object lookup:

```suggestion
type BadgeProps = ComponentProps<typeof Badge>;

const toneForBadge: Record<BadgeLabel, BadgeProps['tone']> = {
  Deprecated: 'caution',
  New: 'positive',
};
```